app/scripts/controllers/landing.js

```diff
@@ -29,23 +29,24 @@ angular.module('confRegistrationWebApp')
     $scope.jumboImg = jumboImgs[_.random(0, jumboImgs.length - 1)];
 
     $scope.eventSearch = _.debounce(function(val) {
-      if(!val){ return; }
-      if(val.length < 2){
+      $scope.loadingMsg = 'Searching Events...';
+      if (!val || val.length < 2) {
+        $scope.loadingMsg = '';
         return;
       }
       $http.get('conferences', {
         params: {
           conferenceName: val
         }
       }).then(function(response){
+        $scope.loadingMsg = '';
         if(angular.isDefined($scope.searchVal) && val === $scope.searchVal) {
```

Comment:
Disabling the loading indicator should be done in here cuz that's where we know we have results for the current search query and that `$scope.searchResults` has been set.